mijia-homie/src/main.rs

```diff
@@ -283,6 +426,40 @@ struct SensorState {
     min_update_period: Duration,
 }
 
+impl SensorState {
```

Comment:
I regret not calling this struct SensorStates (initially I read it as the state of a single sensor and wondered why it would have a `add_sensor_if_named()` method.